gensim/models/wrappers/fasttext.py

```diff
@@ -298,8 +299,16 @@ def load_dict(self, file_handle, encoding='utf8'):
                 char_byte = file_handle.read(1)
             word = word_bytes.decode(encoding)
             count, _ = self.struct_unpack(file_handle, '@qb')
-            assert self.wv.vocab[word].index == i, 'mismatch between gensim word index and fastText word index'
-            self.wv.vocab[word].count = count
+
+            if word != "__label__":
+                self.wv.vocab[word] = Vocab(index=i, count=count)
+                self.wv.index2word.append(word)
+
+        assert len(self.wv.vocab) == nwords, 'mismatch between vocab sizes'
+        if len(self.wv.vocab) != vocab_size:
+            logger.warning("If you are loading any model other than pretrained vector wiki.fr, ")
```

Comment:
Any particular reason for two separate warning statements? Why not a single one?